forge-gui/res/cardsfolder/upcoming/case_of_the_crimson_pulse.txt

```diff
@@ -0,0 +1,14 @@
+Name:Case of the Crimson Pulse
+ManaCost:2 R
+Types:Enchantment Case
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigLoot | TriggerDescription$ When this Case enters the battlefield, discard a card, then draw two cards.
+SVar:TrigLoot:DB$ Discard | Mode$ TgtChoose | SubAbility$ DBDraw
+SVar:DBDraw:DB$ Draw | NumCards$ 2 
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | Hellbent$ True | TriggerZones$ Battlefield | Execute$ SolveFalcon | TriggerDescription$ To solve — You have no cards in hand. (If unsolved, solve at the beginning of your end step.)
```

Comment:
I know it's minor but I'd prefer not reusing SVar names that were tailored for a different card
For this one `TrigSolve` or something would be fine